src/dodal/beamlines/i22.py

```diff
@@ -57,6 +58,32 @@ def it(
     )
 
 
+def vfm(
+    wait_for_connection: bool = True,
+    fake_with_ophyd_sim: bool = False,
+) -> FocusingMirror:
+    return device_instantiation(
+        FocusingMirror,
+        "vfm",
+        "-OP-KBM-01:VFM:",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+    )
+
+
+def hfm(
+    wait_for_connection: bool = True,
+    fake_with_ophyd_sim: bool = False,
+) -> FocusingMirror:
+    return device_instantiation(
+        FocusingMirror,
+        "hfm",
+        "-OP-KBM-01:HFM:",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+    )
```

Comment:
Should: Please could you also make versions on these in the P38 config with `fake_with_ophyd_sim: bool = True`? This allows us to produce documents from P38 with enough info to make a nexus file.